Source/WebCore/platform/graphics/cg/ImageBufferIOSurfaceBackend.cpp

```diff
@@ -156,6 +156,8 @@ unsigned ImageBufferIOSurfaceBackend::bytesPerRow() const
 
 void ImageBufferIOSurfaceBackend::invalidateCachedNativeImage() const
 {
+    if (!m_mayHaveOutstandingBackingStoreReferences)
+        return;
```

Comment:
Is this change relevant to this patch?

Reply:
I believe this was added due to another change.